cluster-autoscaler/cloudprovider/azure/azure_agent_pool.go

```diff
@@ -302,6 +303,10 @@ func (as *AgentPool) IncreaseSize(delta int) error {
 	as.mutex.Lock()
 	defer as.mutex.Unlock()
 
+	if as.curSize == -1 {
+		return fmt.Errorf("the availability set %s is under initialization, skipping the operation", as.Name)
```

Comment:
skipping the "increase size" or "scale up" operation - whatever works  so its easier to figure out from logs without having to look at the source code